tools/cockpit.spec

```diff
@@ -155,8 +162,13 @@ echo '{ "linguas": null, "machine-limit": 5 }' > %{buildroot}%{_datadir}/%{name}
 echo '%dir %{_datadir}/%{name}/base1' > base.list
 find %{buildroot}%{_datadir}/%{name}/base1 -type f >> base.list
 
+%if %{defined build_dashboard}
 echo '%dir %{_datadir}/%{name}/dashboard' >> dashboard.list
 find %{buildroot}%{_datadir}/%{name}/dashboard -type f >> dashboard.list
+%else
+rm -rf %{buildroot}/%{_datadir}/%{name}/dashboard
+touch dashboard.list
```

Comment:
Not sure, but this is how we currently deal with docker and kubernetes as well: https://github.com/cockpit-project/cockpit/blob/11f88d12543a94edf0841053f82fca66a9f409af/tools/cockpit.spec#L209
It seems like a build-system level decision to do this, e.g. it may also depend on the architecture.